modules/fbx/editor_scene_importer_fbx.cpp

```diff
@@ -180,21 +180,23 @@ Node *EditorSceneImporterFBX::import_scene(const String &p_path, uint32_t p_flag
 				}
 			}
 
-			if (!is_blender_fbx) {
-				Spatial *spatial = _generate_scene(p_path, &doc, p_flags, p_bake_fps, 8);
-				// todo: move to document shutdown (will need to be validated after moving; this code has been validated already)
-				for (FBXDocParser::TokenPtr token : tokens) {
-					if (token) {
-						delete token;
-						token = nullptr;
-					}
-				}
+			if (is_blender_fbx) {
+				WARN_PRINT("We officially don't support blender FBX animations yet, due to issues with upstream blender, \n"
+						   "so please wait for us to add this in. We will continue to import the file but it may be broken.\n"
+						   "for minimal breakage please export fbx from blender with -Z forward, and Y up.");
```

Comment:
```suggestion
				WARN_PRINT("We don't officially support Blender FBX animations yet, due to issues with upstream Blender,\n"
						   "so please wait for us to work around remaining issues. We will continue to import the file but it may be broken.\n"
						   "For minimal breakage, please export FBX from Blender with -Z forward, and Y up.");
```